src/text_mate_tools/preprocess_document_rules.py

```
@@ -234,46 +216,50 @@ async def main():
 
     total_time = time.time() - total_start_time
 
-    print("\n✨ Process complete!")
+    print("\n✨ Extraction complete!")
     print(f"📊 Documents processed: {documents_processed}/{len(document_paths)}")
-    print(f"📊 Total rules saved: {total_rules_saved}")
+    print(f"📊 Total rules extracted: {total_rules_saved}")
     print(f"⏱️ Total processing time: {total_time:.2f}s")
-    print(f"💾 Rules directory: {rules_dir.absolute()}")
+    print(f"💾 Staging directory: {rules_dir.absolute()}")
+    print(f"\n👉 Next steps:")
```

Ruff (0.15.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)